Userland/Libraries/LibCore/DirIterator.cpp

```diff
@@ -9,6 +9,10 @@
 #include <errno.h>
 #include <unistd.h>
 
+#if defined(AK_OS_WIN32)
+#    include <dirent.h>
```

Comment:
eventually if you would want to do native win32 you would need [fileapi.h](https://docs.microsoft.com/en-us/windows/win32/fileio/file-management-functions?redirectedfrom=MSDN)
however that is not going to be as clean as what you have here. I doubt you would want a bunch of ifdefs for windows. Instead I would think [c++17's filesystem](https://en.cppreference.com/w/cpp/header/filesystem) api would be a better crossplat solution longterm

Reply:
The entire point of Lagom (and Serenity in general) is to _not_ have to use the STL, and instead implement everything locally!